src/store/Selectors/GraphData.ts

```diff
@@ -100,14 +100,14 @@ export const decorateGraphData = (graphData: GraphElements): DecoratedGraphEleme
         if (decoratedNode.data.traffic) {
           const traffic = decoratedNode.data.traffic;
           decoratedNode.data.traffic = undefined;
-          traffic.map(protocol => {
+          traffic.forEach(protocol => {
             decoratedNode.data = { ...protocol.rates, ...decoratedNode.data };
           });
         }
         // prettier-ignore
```

Comment:
Does this mean that we can now get rid of the `// prettier-ignore` now?

Reply:
I think so, we can do that on another PR.